_drafts/2018-08-14-docviewer.md

```diff
@@ -0,0 +1,53 @@
+Since we wanted an app with mostly viewing capabilities and a little bit of editing qualities, Apple preview was the closest to our concept.
+
+After clarifying the main path that we should take, I conducted a few user interviews (six to be exact) to find out what people mainly looknig for when they open a document viewer. After the interviews were done and evaluated each interviewee's answers I began to make the wireframes. After several versions there was a clear winner among the team and from there the prototyping began.
+
+![Wireframes](/img/posts/Wireframe.png "Selected Wireframes")
+
+## Prototypes
+After the wireframing I made high fidelity prototypes in light and dark mode. For the high fidelity prototype I used [Sketch](https://www.sketchapp.com/) and followed [Material Design](https://material.io/design/) principles. I also made a click through prototype in [InVision](https://www.invisionapp.com/) which you can try as well in the following link: [Docviewer](https://projects.invisionapp.com/share/TGJZD5KNQZU#/screens)
+
+![Prototype1](/img/posts/Docviewer-Active-Light.png "Prototype light")
+
+![Prototype2](/img/posts/Docviewer-Active-Dark.png "Prototype dark")
+
+
+## User testing
+When the click through prototype was finished I did a user testing session which included 5 people.
+I asked the participants to follow through these steps:
+1: Open the edit feature
+2: Open the left sidebar
+3: Go to gage 2 and 3 then go back to the top of the page
+4: Open the comment section
+5: Zoom in and out
+6: Rotate the page to the left and then back to it's original state
+
+
+## Iteration
+After finishing the user tests I evaluated the answers and there was a few things that needed to be changed. First of all the left sidebar icon was very unfamiliar for most of the participants and had a hard time finding it, so I had to find a more straightforward icon. Most participants liked the comment function and said it would benefit them. Also most of the participants didn't liked that the rotate buttons were on the bottom of the page, so I moved it to the top navbar.
+
+![Iteration](/img/posts/Docviewer_mockup.png "Iterated design")
+
+## User testing part2
+I updated my design after the first set of user tests and I conducted a second round test with the same participants. The results were much better this time and eveyone made through the tests without any problem. The new sidebar icon solved the problem and putting the rotate buttons to the top navbar helped a lot as well. I also changed the order of the icons in the top navbar so the zoom icons were next to the rotate and sidebar, download, print and share buttons were next to each other.
```

Comment:
this time and eveyone --> this time and everyone

maybe to explain the order of the button you could import a screenshot of the navbar (before & after)
--> I also changed the order of the icons as following: